modules/balana-utils/pom.xml

```diff
@@ -32,19 +32,23 @@
     </dependencies>
 
     <build>
+        <pluginManagement>
```

Comment:
Otherwise I would get an error message in the pom.xml saying phase incremental-build does not exist.